pupil-spa/src/app/services/question/question.service.ts

```diff
@@ -14,7 +14,7 @@ export class QuestionService {
   protected config: Config;
 
   constructor(protected storageService: StorageService) {
-    this.currentQuestion = 1;
+    this.currentQuestion = 0;
```

Comment:
‼️  Don't like this at all.   Questions are always numbered from 1 .. n 
Please don't change this.

Reply:
Thought I reverted that.
@jon-shipley it's updated